core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/memory/HeapDumpRule.java

```diff
@@ -0,0 +1,75 @@
+package org.openjdk.jmc.flightrecorder.rules.jdk.memory;
```

Comment:
Needs copyright notice. Copy one which includes Datadog, ensure year is set to 2020. (The Datadog part is not added retroactively to files initiated by Oracle, but for entirely new files, like this one, we should add it.)